.editorconfig

```diff
@@ -0,0 +1,28 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+tab_width = 4
+
+[*.gradle]
+indent_style = tab
+
+[*.java]
+indent_style = tab
+max_line_length = off
+ij_continuation_indent_size = 8 # IntelliJ default
+# Wrapping and Braces
+ij_java_keep_simple_blocks_in_one_line = true
+ij_java_keep_simple_classes_in_one_line = true
+ij_java_keep_simple_lambdas_in_one_line = true
+ij_java_keep_simple_methods_in_one_line = true
```

Comment:
this means that a method such as:
```java
public static int square(int a) {
    return a * a;
}
```
is formatted like that and not
```java
public static int square(int a) { return a * a; }
```
right?

Reply:
Yes, it won’t turn the first one into the second one. But I think the second one might be allowed too. It’s intended to allow empty blocks in one line like these:
```java
@Override
public void reset() {}
```
or else a new line gets added between the brackets.